src/Form.js

```diff
@@ -0,0 +1,254 @@
+import React, { Component } from 'react'
+import './App.css'
+
+class Form extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      name: '',
+      email: '',
+      pass: '',
+      avatar: '',
+      phone: '',
+      age: 1,
+      gender: 'male',
+      favorite: {
+        soccer: false,
+        badminton: false,
+        sailing: false
+      },
+      note: "Buổi học đầu tiên về html"
+    }
+  }
+
+  handleChangeName = event => {
+    this.setState({
+      name: event.target.value
+    });
+  }
+
+  handleChangeEmail = event => {
+    this.setState({
+      email: event.target.value
+    });
+  }
+
+  handleChangePass = event => {
+    this.setState({
+      pass: event.target.value
+    });
+  }
+
+  handleChangeAvatar = event => {
+    this.setState({
+      avatar: event.target.value
+    });
+  }
+
+  handleChangePhone = event => {
+    this.setState({
+      phone: event.target.value
+    });
+  }
+
+  handleChangeAge = event => {
+    this.setState({
+      age: event.target.value
+    });
+  }
+
+  handleChangeGender = event => {
+    this.setState({
+      gender: event.target.value
+    });
+  }
+
+  handleChangeNote = event => {
+    this.setState({
+      note: event.target.value
+    });
+  }
+
+  handleChangeSoccer = event => {
+    this.setState(prevState => ({
+      soccer: !prevState.soccer,
+    }));
+  }
+
+  handleChangeBadminton = event => {
+    this.setState(prevState => ({
+      badminton: !prevState.badminton,
+    }));
+  }
+
+  handleChangeSailing = event => {
+    this.setState(prevState => ({
+      sailing: !prevState.sailing,
+    }));
+  }
+
+  handleSubmit = event => {
+    const favorites = {
+      soccer: this.state.soccer,
+      badminton: this.state.badminton,
+      sailing: this.state.sailing
+    }
+    const object = {
+          name : this.state.name,
+          phone : this.state.phone,
+          email : this.state.email,
+          pass : this.state.pass,
+          avatar : this.state.avatar,
+          age : this.state.age,
+          gender : this.state.gender,
+          favorite: favorites,
+          note : this.state.note }
+    event.preventDefault();
+    console.log(object)
```

Comment:
Chỗ thì ghi có `;` chỗ thì không có. Nếu muốn ghi `;` thì dòng nào cũng phải ghi. Nếu không ghi `;` thì phải xóa hết toàn bộ `;` trong project đi